com.unity.render-pipelines.high-definition/Runtime/Lighting/ProbeVolume/ProbeVolume.hlsl

```diff
@@ -1,6 +1,7 @@
 #ifndef __PROBEVOLUME_HLSL__
 #define __PROBEVOLUME_HLSL__
 
+#include "Packages/com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs.hlsl"
```

Comment:
shouldn't be required, shaderconfig.hlsl is include in shadervariables.hlsl that is always include before other stuff.